op-mode-definitions/show-configuration.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
+<interfaceDefinition>
+  <node name="show">
+    <children>
+    <node name="configuration">
+      <properties>
+        <help>Show available saved configurations</help>
+      </properties>
+      <!-- no admin check -->
+      <command>cli-shell-api showCfg --show-active-only --show-hide-secrets</command>
+      
+      <children>
+        <node name="all">
+          <properties>
+            <help>Show running configuration (including default values)</help>
+          </properties>
+          <!-- no admin check -->
+          <command>cli-shell-api showCfg --show-show-defaults --show-active-only --show-hide-secrets</command>
+        </node>
+        <node name="commands">
+          <properties>
+            <help> Show running configuration as set commands </help>
+          </properties>
+          <!-- no admin check -->
+          <command>${vyatta_sbindir}/vyatta-config-gen-sets.pl</command>
```

Comment:
vyatta-config-gen-sets.pl is no more. I've replaced it with a better implementation, see https://phabricator.vyos.net/T684